src/Compilers/Core/Portable/Operations/Loops/IForEachLoopStatement.cs

```diff
@@ -14,13 +14,20 @@ namespace Microsoft.CodeAnalysis.Semantics
     public interface IForEachLoopStatement : ILoopStatement
```

Comment:
>IForEachLoopStatement [](start = 21, length = 21)

It looks like we are not exposing information about whether conversion applied to the ```Current``` is checked. Should we?

Reply:
Information about conversion is not exposed as well. Please open an issue to follow up.

---
In reply to: [137680570](https://github.com/dotnet/roslyn/pull/21791#discussion_r137680570) [](ancestors = 137680570)

Reply:
Filed https://github.com/dotnet/roslyn/issues/22021